homeassistant/components/lutron_caseta/diagnostics.py

```diff
@@ -21,11 +21,18 @@ async def async_get_config_entry_diagnostics(
             "title": entry.title,
             "data": dict(entry.data),
         },
-        "data": {
+        "bridge_data": {
             "devices": bridge.devices,
             "buttons": bridge.buttons,
             "scenes": bridge.scenes,
             "occupancy_groups": bridge.occupancy_groups,
             "areas": bridge.areas,
         },
+        # "integration_data": {
+        #    "keypads": data.keypads,
+        #    "keypad_buttons": data.keypad_buttons,
+        #    "keypad_button_names_to_leap": data.keypad_button_names_to_leap,
+        #    "keypad_trigger_schemas": data.keypad_trigger_schemas,
+        #    "dr_device_id_to_keypad": data.dr_device_id_to_keypad,
+        # },
```

Comment:
test data?

Reply:
Yes, but it should probably still be there.  It's useful.
Maybe I'll update the tests so I can add it back in.

Reply:
working on modifying the tests for this.
Having 1 problem.  Part of the output is the DeviceRegistry device ID.  But this changes everytime I run the tests.  Is there a way to exclude part of it output of data.keypad_data.keypads?

```
     assert diag == { <diag output> }

        "integration_data": {
            "dr_device_id_to_keypad": {
                "a363c3043e0cb46ef19891a361146273": {
'''

Reply:
Its ok to mock it as `ANY` via `from unittest.mock import ANY`

Reply:
Figuring this mock stuff out will be a side project for me.  I've just removed the commented out diagnostic items for now.

Reply:
ok, I figured it out.  I was able to patch device_registry.async_get_or_create() to return a predictable device id.
The data has been added back into the diagnostics, and tests updated